suite-native/module-transactions/src/components/TransactionList.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { ActivityIndicator } from 'react-native';
+
+import { A } from '@mobily/ts-belt';
+
+import { useNativeStyles } from '@trezor/styles';
+import { WalletAccountTransaction } from '@suite-common/wallet-types';
+import { groupTransactionsByDate } from '@suite-common/wallet-utils';
+
+import { TransactionListItem } from './TransactionListItem';
+import { TransactionListGroup } from './TransactionListGroup';
+
+type AccountTransactionProps = {
+    transactions: WalletAccountTransaction[];
+};
+
+export const TransactionList = ({ transactions }: AccountTransactionProps) => {
+    const { utils } = useNativeStyles();
+    const accountTransactionsByDate = groupTransactionsByDate(transactions);
+    const transactionDateKeys = Object.keys(accountTransactionsByDate);
+
+    if (A.isEmpty(transactionDateKeys))
```

Comment:
Also if there will be no transactions for account so far, there will be displayed loading forever? That's not good, at least create task for it.

Reply:
Well that should never happen because suite doesn't let you create empty account - if there are no transactions, it gets deleted. So from xpub, this shouldn't be possible. 

But yeah, it could probably be handled better. https://github.com/trezor/trezor-suite/issues/6251